### ISSUE TITLE
JSX Renderer: defaults `docType` as `true`

### ISSUE THREAD

**Issue by yusukebe** [honojs/hono#2052]
### What is the feature you are proposing?

I think we should just make `default:true` the default, since it's a pain to write `docType:true` every time.

```ts
export default jsxRenderer(
  ({ children, title }) => {
    return (
      <html lang='en'>
        <head>
          <meta charset='UTF-8' />
          <meta name='viewport' content='width=device-width, initial-scale=1.0' />
        </head>
        <body>{children}</body>
      </html>
    )
  },
  {
    docType: true, // make it as default
  }
)
```

**Comment by EdamAme-x** [comment]
https://github.com/honojs/hono/pull/2064

**Comment by yusukebe** [comment]
Fixed by #2064
